pages/docs/tutorials/configure-asyncapi-kafka.md

```diff
@@ -0,0 +1,161 @@
+
+[Publish/Subscribe (Pub/Sub)](/docs/tutorials/getting-started/event-driven-architectures#publishersubscriber) is a typical model in EDAs, providing flexible coupling between publishers and subscribers. In this model, the publishers of event notifications act as publishers and the consumers act as corresponding subscribers.
+
+## Creating AsyncAPI document for Kafka
+
+In this section, you’ll create an AsyncAPI document to describe the `UserSignUp` API. The same document can be later used to generate code and documentation as per requirement. 
+
+### Define AsyncAPI version, API information, and server
+
+Initially, you need to describe your application, including the AsyncAPI version, the info about the document, and the server your application is based upon.
+
+```
+asyncapi: 3.0.0
+info:
+  title: User Signup API
+  version: 1.0.0
+  description: The API notifies you whenever a new user signs up in the application.
+servers:
+  kafkaServer:
+    host: test.mykafkacluster.org:8092
+    description: Kafka Server
+    protocol: kafka
+```
+
+In the above snippet:
+
+- The `asyncapi` field indicates that you are using AsyncAPI version 3.0.0.
+  
+- The `info` field provides information about the API. Here the the APIs `title`, `version`, and `description` are being defined.
+  
+- The `server` field specifies the details of the server, including the `host`, `description`, and the `protocol` that is being used i.e. Kafka.
+
+### Define channels and operations
+
+Next, let's move on to the `channels` and `operations` section. The channel addresses are the topics in Kafka, they are the routes to which your API will be sending/receiving. The `operations` section is used to describe how your application interacts with the channels.
+
+```
+operations:
+  onUserSignedUp:
+    action: receive
+    channel:
+      $ref: '#/channels/userSignedUp'
+
+
+channels:
+  userSignedUp:
+    description: This channel contains a message per each user who signs up in our application.
+    address: user_signedup
+    messages:
+      userSignedUp:
+        $ref: '#/components/messages/userSignedUp'
+```
+
+In the above snippet:
+
+- The `onUserSignedUp` object specifies the nature of the operation. The `action` property indicates that our application will be receiving the information. The `channel` property points to the channel where the operation occurs.
+  
+- The `userSignedUp` object inside `channels` describes the Kafka topic where our application will be receiving the information and the associated message definition. The `address` field represents the actual name of the Kafka topic. The `messages` field describes the expected messages in that topic. 
+
+### Define messages and its schemas
+
```

Comment:
Ooops, heyooo @J0SAL, it looks like this needs to be updated to use **Avro** schema.